code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -41,8 +41,10 @@
 			stacktype = material.stack_type
 
 /obj/item/stack/tile/attackby(obj/item/W as obj, mob/user as mob)
-	..()
 	if (istype(W, /obj/item/weldingtool))
+		// do the thing
+		return TRUE
+	return ..()

```

Comment:
Errr....

Reply:
spent the last five minutes looking for // do the thing in the code before realising you meant thats where the code went smh